Resources/en.lproj/Localizable.strings

```diff
@@ -1,11 +1,15 @@
 "General_Calibration" = "Calibration: %d%%";
 
 "Welcome_Title" = "Welcome to Mapbox Vision Teaser!";
-"Welcome_Body" = "I have read and agree, on behalf of myself and my\ncompany and its affiliates, to be bound by the terms of\nthe Vision SDK Evaluation Agreement.";
-"Welcome_Body_LinkSubstring" = "Evaluation Agreement";
+"Welcome_Body" = "By using this application you acknowledge that you have read and agree to the Mapbox Terms of Service and Privacy Policy";
+"Terms_of_Service_Option" = "Terms of Service";
```

Comment:
I was considering the following scheme of localization keys: (context where the key is used, typically the name of the screen, or `General` if used in multiple cases)\_(type of element e.g. title / body / button)\_(specification if there's more than one instances of that type). Not really following this pattern for menu buttons but will adapt all the keys in the scope of https://github.com/mapbox/vision-ios-examples/issues/82.

This key is used as both substring and option so consider moving it to general section with a name like `General_TermsOfService`. (we may agree on the format first in #82).